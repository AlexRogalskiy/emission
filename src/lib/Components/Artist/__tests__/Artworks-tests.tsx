import "react-native"

import React from "react"
import * as renderer from "react-test-renderer"

import Artworks from "../ArtistArtworks"

import { Theme } from "@artsy/palette"

it("renders properly", () => {
  const artist = {
    counts: {
      artworks: 5,
      for_sale_artworks: 2,
    },
  }
  const artworks = renderer
    .create(
      <Theme>
        <Artworks artist={artist as any} />
      </Theme>
    )
    .toJSON()
  expect(artworks).toMatchSnapshot()
})
