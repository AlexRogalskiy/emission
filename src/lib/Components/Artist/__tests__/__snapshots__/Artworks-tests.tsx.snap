// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders properly 1`] = `
<ArtistArtworks
  artist={
    Object {
      "counts": Object {
        "artworks": 5,
        "for_sale_artworks": 2,
      },
    }
  }
/>
`;
